### PR TITLE
Update ntldr.h,ntrtl.h

### PR DESCRIPTION
**Pull request by MiroKaku**

### CODE REVIEW

**Comment by MiroKaku** [comment]
I have read the CLA Document and I hereby sign the CLA

File: phnt/include/ntldr.h [hunk 1]
```diff
@@ -297,6 +297,17 @@ LdrGetDllFullName(
...
+// rev
+NTSYSAPI
+NTSTATUS
+NTAPI
+LdrGetDllPath(
+    _In_  PCWSTR DllName,
+    _In_  ULONG  Flags, // LOAD_LIBRARY_SEARCH_*
+    _Out_ PWSTR* DllPath,
+    _Out_ PWSTR* SearchPaths
+);
```
Comment (jxy-s) [inline phnt/include/ntldr.h:309]:
```suggestion
    );
```

File: phnt/include/ntrtl.h [hunk 2]
```diff
@@ -3280,6 +3280,18 @@ RtlReleaseActivationContext(
...
+NTSYSAPI
+NTSTATUS
+NTAPI
+RtlFindActivationContextSectionString(
+    _In_ ULONG Flags,
+    _In_ const GUID* ExtensionGuid,
+    _In_ ULONG SectionId,
+    _In_ const UNICODE_STRING* StringToFind,
+    _Out_ PACTCTX_SECTION_KEYED_DATA ReturnedData
+);
```
Comment (jxy-s) [inline phnt/include/ntrtl.h:3293]:
```suggestion
    );
```

File: phnt/include/ntrtl.h [hunk 3]
```diff
@@ -3818,6 +3830,22 @@ RtlDoesFileExists_U(
...
+    _In_ ULONG                  Flags,
+    _In_ PUNICODE_STRING        OriginalName,
+    _In_ PUNICODE_STRING        Extension,
+    _In_opt_ PUNICODE_STRING    StaticString,
+    _In_opt_ PUNICODE_STRING    DynamicString,
+    _In_opt_ PUNICODE_STRING*   NewName,
+    _In_ PULONG                 NewFlags,
+    _In_ PSIZE_T                FileNameSize,
+    _In_ PSIZE_T                RequiredLength
+);
```
Comment (jxy-s) [inline phnt/include/ntrtl.h:3847]:
```suggestion
    );
```

**Review by jxy-s — CHANGES_REQUESTED** [review]
I confirmed the additions and corrections myself. Please make the formatting adjustments.

**Review by jxy-s — APPROVED** [review]

**Pull request merged**